packages/core/CalendarManager.ts

```diff
@@ -206,7 +206,7 @@ export const getBusyCalendarTimes = async (
   selectedCalendars: SelectedCalendar[]
 ) => {
   let results: EventBusyDate[][] = [];
-  // const months = getMonths(dateFrom, dateTo);
+  const months = getMonths(dateFrom, dateTo);
```

Comment:
It is currently unused and not doing anything after uncommenting. How did this fix the current issue?